include/miopen/miopen.h

```diff
@@ -72,6 +72,7 @@
  * @defgroup ReduceCalculation
  * @defgroup RotaryPositionalEmbeddings
  * @defgroup ReLU
+ * @defgroup cartesianprod
```

Comment:
Let use group `LossFunction` instead. It is already there, you just have to remove this line.